module_3/task3.js

```diff
@@ -0,0 +1,42 @@
+import csvtojson from "csvtojson"
+import fs from "fs"
+const CSV_FILE_PATH = ".csv/data-csv.csv"
+
+async function fromCSVtoTXT() {
+  try {
+    const onError = (err) => {
+      console.error(err)
+    }
+
+    const onComplete = () => {
+      console.log("completed")
+    }
+
+    csvtojson()
+      .fromFile(CSV_FILE_PATH)
```

Comment:
using `fromFile` is loading the full file into memory which is not requested in this task, the idea is reading line by line using streamings, check `csvtojson` documentation to see how you could refactor it